repo/packages/algebra-plugins/package.py

```diff
@@ -0,0 +1,37 @@
+from spack.package import *
+
+
+class AlgebraPlugins(CMakePackage):
+    homepage = "https://github.com/acts-project/algebra-plugins"
+    git = "https://github.com/acts-project/algebra-plugins.git"
+    list_url = "https://github.com/acts-project/algebra-plugins/tags"
+
+    version("0.17.0", commit="f4e9c1cc37ce6080234f265d121f4bc0a74bef87")
+
+    variant("vc", default=False, description="Enables the Vc plugin")
+    variant("eigen", default=False, description="Enables the Eigen plugin")
+    variant("vecmem", default=False, description="Enables the vecmem plugin")
```

Comment:
Since the CMakeLists for this project manually fiddles with the C++ standard, I would advise adding a `cxxstd` variant here, [in the spirit of what the root package does](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/root/package.py#L194). People around here (including you) are already experimenting with C++20, and since support for it in GCC and clang is currently quite brittle, sticking with C++17 is also a very valid option, so given that there are two valid choices for this build parameter, there should be a variant exposing the choice.

This also applies to the detray package, which similarly manually fiddles with CMAKE_CXX_STANDARD.

Reply:
Also, you will want to add support for the other algebra-plugins build options. And for readability/maintainability, alphabetical order in the lists of variants and dependencies would be nice. This is a remark that generally applies to your other packages.